common-gradle-enterprise-gradle-configuration/settings.gradle.kts

```diff
@@ -0,0 +1,36 @@
+plugins {
+    id("com.gradle.enterprise") version("3.11.1")
+    id("com.gradle.common-custom-user-data-gradle-plugin") version("1.8")
+}
+
+val isCI = !System.getenv("CI").isNullOrEmpty() // adjust to your CI provider
+
+gradleEnterprise {
+    server = "https://enterprise-samples.gradle.com" // adjust to your GE server
+    allowUntrustedServer = false // ensure a trusted certificate is configured
```

Comment:
This looks like it would be `true` by default, which is not the case

Reply:
I agree, I just took the sample as it was from `settings.gradle.kts`. @runningcode shouldn't we instead have `allowUntrustedServer = true` commented out ?

Reply:
Our examples should propose secury config, not insecure config. Thus, I would not set this to true.

Whether setting it to false implies that the default value is true does not seem a major point to me, and I would stick to explicitly setting it to false so people understad that the option exists.